crates/as-if-std/Cargo.toml

```diff
@@ -29,7 +29,7 @@ features = ['read_core', 'elf', 'macho', 'pe', 'xcoff', 'unaligned', 'archive']
 [build-dependencies]
 # Dependency of the `backtrace` crate
 cc = "1.0.97"
-windows-bindgen = "0.56"
+windows-bindgen = "0.58"
```

Comment:
Updating `windows-bindgen` should also update the generated bindings, which involves running the `riddle` tool (`cargo install riddle`).

Reply:
I removed this update in this PR, should probably be done separatelly